File: TaskSystemCli/DataCenter/Roles.cs
using Newtonsoft.Json;
using System.Collections.Generic;
using System.IO;
using System.Threading.Tasks;

namespace TaskSystemCli.DataCenter
{
    class Roles
    {
        private static readonly string Address = "DataCenter/Roles.json";

        public string Emoji { get; set; }
        public string FullName { get; set; }
        public string[] Kinders { get; set; }
        public string ShortName { get; set; }
        public string Team { get; set; }
        public string[] Achivments { get; set; }

        public static async Task<List<Roles>> GetRolesAsync()
        {
            using StreamReader sr = new StreamReader(Address);
            return JsonConvert.DeserializeObject<List<Roles>>(await sr.ReadToEndAsync());
        }
    }
}

File: task-cli-python/data/roles.py
import json
from typing import List

class role:
    def __init__(
        self, 
        emoji, 
        fullName, 
        kinders, 
        short_name, 
        team, 
        achivments
    ):

        self.emoji = emoji
        self.fullName = fullName
        self.kinders = kinders
        self.short_name = short_name
        self.team = team
        self.achivments = achivments

    @staticmethod
    def get_roles():
        with open("data/roles.json", 'r', encoding="utf8") as f:
            datastore = json.load(f)
        
        result = []
        for r in datastore:
            result.append(
                role(
                    r["Emoji"],
                    r["FullName"],
                    r["Kinders"],
                    r["ShortName"],
                    r.get("Team", None),
                    r.get("Achivments", None)
                )
            )
            
        return result

